py/client-ticking/src/pydeephaven_ticking/_core.pxd

```diff
@@ -128,45 +128,45 @@ cdef extern from "deephaven/dhcore/column/column_source_helpers.h" namespace "de
 
 cdef extern from "deephaven/dhcore/types.h" namespace "deephaven::dhcore":
     ctypedef enum ElementTypeId "deephaven::dhcore::ElementTypeId::Enum":
-        CHAR "deephaven::dhcore::ElementTypeId::CHAR"
-        INT8 "deephaven::dhcore::ElementTypeId::INT8"
-        INT16 "deephaven::dhcore::ElementTypeId::INT16"
-        INT32 "deephaven::dhcore::ElementTypeId::INT32"
-        INT64 "deephaven::dhcore::ElementTypeId::INT64"
-        FLOAT "deephaven::dhcore::ElementTypeId::FLOAT"
-        DOUBLE "deephaven::dhcore::ElementTypeId::DOUBLE"
-        BOOL "deephaven::dhcore::ElementTypeId::BOOL"
-        STRING "deephaven::dhcore::ElementTypeId::STRING"
-        TIMESTAMP "deephaven::dhcore::ElementTypeId::TIMESTAMP"
+        kChar "deephaven::dhcore::ElementTypeId::kChar"
+        kInt8 "deephaven::dhcore::ElementTypeId::kInt8"
+        kInt16 "deephaven::dhcore::ElementTypeId::kInt16"
+        kInt32 "deephaven::dhcore::ElementTypeId::kInt32"
+        kInt64 "deephaven::dhcore::ElementTypeId::kInt64"
+        kFloat "deephaven::dhcore::ElementTypeId::kFloat"
+        kDouble "deephaven::dhcore::ElementTypeId::kDouble"
+        kBool "deephaven::dhcore::ElementTypeId::kBool"
+        kString "deephaven::dhcore::ElementTypeId::kString"
+        kTimestamp "deephaven::dhcore::ElementTypeId::kTimestamp"
```

Comment:
Is this some sort of standard?  If not, I liked the prior better.

Reply:
I agree, it's uglier than [REDACTED] but https://google.github.io/styleguide/cppguide.html#Constant_Names